src/template.js

```diff
@@ -114,7 +114,9 @@ export const TemplateChunk = Object.freeze({
 
       if (mustRemoveRoot && this.el.parentNode) {
```

Comment:
Logically this `if` can go inside the next, putting it there is logically equivalent and avoids the check when `mustRemoveRoot` is null